try:
    import try_except
except SystemError:
    print('OK')